README.rst

```diff
@@ -135,6 +135,12 @@ You can also use another modules as numpy or matplotlib to work with beprof::
     plt.plot(foo[:,0], foo[:,1])
     plt.show()
 
+
+Note that beprof is a library and end-users shouldn't "run it".
```

Comment:
Mention also that end users are stroungly discouraged to use `git clone` to get the code and run it.
git is only for developers, users should use pip.

Remove also abovementioned information about examples in `feature/examples` branch.